staff/lluis-bagur/cv-styled/styles/styles.css

```diff
@@ -0,0 +1,242 @@
+
+*{
```

Comment:
The general indentation of the file is little bit messy. Maybe it's better to organize it.